File: test/ayush.dev/python/pipelines/PythonInsideProjectDatasetsOnly/code/pythoninsideprojectdatasetsonly/graph/Lookup_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from pythoninsideprojectdatasetsonly.config.ConfigStore import *
from pythoninsideprojectdatasetsonly.udfs.UDFs import *

def Lookup_1(spark: SparkSession, in0: DataFrame):
    keyColumns = ['''customer_id''', '''email''']
    valueColumns = ['''first_name''', '''last_name''']
    createLookup("LookupMain", in0, spark, keyColumns, valueColumns)

File: test/randomtest/scallaexternalgitone/pipelines/SC_CONFIG_EXT_GIT_ONE/code/src/main/scala/io/prophecy/pipelines/sc_config_ext_git_one/graph/Lookup_1.scala
package io.prophecy.pipelines.sc_config_ext_git_one.graph

import io.prophecy.libs._
import io.prophecy.pipelines.sc_config_ext_git_one.config.Context
import io.prophecy.pipelines.sc_config_ext_git_one.udfs.UDFs._
import io.prophecy.pipelines.sc_config_ext_git_one.udfs.PipelineInitCode._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Lookup_1 {

  def apply(context: Context, in0: DataFrame): Unit =
    createLookup("TestLookup",
                 in0,
                 context.spark,
                 List("c_timestamp"),
                 "c-decimal"
    )

}
